core/tests/protractor_utils/ProfilePage.js

```diff
@@ -70,14 +78,20 @@ var ProfilePage = function() {
   };
 
   this.expectUserToHaveInterestPlaceholder = async function(expectedText) {
+    await waitFor.visibilityOf(interestPlaceholder,
+      'interestPlaceholder label element taking too long to appear');
     expect(await interestPlaceholder.getText()).toMatch(expectedText);
   };
 
   this.expectUserToNotHaveInterestPlaceholder = async function() {
+    await waitFor.visibilityOf(interestPlaceholder,
+      'interestPlaceholder label element taking too long to appear');
```

Comment:
ditto: I don't think this will work since we're expecting the placeholder to be missing

Reply:
not done